agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -485,9 +485,10 @@ bool slave_thread::handle_cmdu_control_ieee1905_1_message(Socket *sd,
         return handle_channel_preference_query(sd, cmdu_rx);
     case ieee1905_1::eMessageType::CHANNEL_SELECTION_REQUEST_MESSAGE:
         return handle_channel_selection_request(sd, cmdu_rx);
-    case ieee1905_1::eMessageType::CLIENT_CAPABILITY_QUERY_MESSAGE: {
+    case ieee1905_1::eMessageType::CLIENT_CAPABILITY_QUERY_MESSAGE:
         return handle_client_capability_query(sd, cmdu_rx);
-    }
+    case ieee1905_1::eMessageType::MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE:
+        return handle_multi_ap_policy_config_request(sd, cmdu_rx);
```

Comment:
why add "multi_ap" prefix? we don't do that for the rest of the APIs, so I guess we shouldn't here as well (nitpick)
```suggestion
        return handle_ap_policy_config_request(sd, cmdu_rx);
```

Reply:
"MULTI_AP" prefix is unique to this specific message. We ignore "_MESSAGE" suffix which is common to all message types.
I think it is helpful to have this message since it is easier to find that function when searching for this message type string.